src/runner.ts

```diff
@@ -67,9 +70,12 @@ loadingNetwork.then(([trips, transfers, interchange, calendars]) => {
     const startDate = new Date(req.query.startDate);
     const endDate = new Date(req.query.endDate);
     const searchDate = getMidnight(startDate.toISOString());
-    console.log(orig, dest, searchDate, startDate, endDate);
 
-    const journeys = detailedQuery.plan(orig, dest, searchDate, startDate, endDate);
+    const notVias = req.query.notVia ? req.query.notVia : [];
```

Comment:
We should sanity check the origin and destination against the notVia list, I have a feeling that this will break the algorithm further down.

Reply:
Also, it means we can reply with a sane error message instead of perhaps an empty result!

Reply:
8c8b9d7 ive added checks to this in runner.ts but i was wondering if there should be additional checks in the raptorAlgortithm it self since any future uses wouldnt guard against this. Do you think its worth it throwing an Error in both scan methods?

Reply:
I have mixed feelings about throwing an error in the scans. 
It's nice because you can keep that logic in one place. On the other hand, there's no way to signal that this will throw an error in the code nicely. 

I'm sure it's not idiomatic JS/TS to expect to handle errors like that. Usually, there's an error callback that is passed in, and the function usually calls the error function if it errors. 

In this case, I'd also argue that we shouldn't throw an error, we should return something more helpful. We should return a `400` response at the very least!

Reply:
7fb6b84 corrected to send a 400 response when it finds the origin or destination in the not via list.
 Body of message will just the message. E.g: Origin: BTN in not via list: [BTN]

Logs:
{
  orig: 'BTN',
  dest: 'LBG',
  startDate: '2019-04-29T10:00:00',
  notVia: '[BTN]'
}
Origin: BTN in not via list: [BTN]
{
  orig: 'BTN',
  dest: 'LBG',
  startDate: '2019-04-29T10:00:00',
  notVia: '[LBG]'
}
Destination: LBG in not via list: [LBG]

Reply:
Cool.